### DISCUSSION TITLE
`extraVolumes` in Woodpecker Agent.

### DISCUSSION THREAD

**Discussion by GamePlayer-8** [woodpecker-ci/woodpecker#3241] [Ask a Question]
Good afternoon, 

How to use `extraVolumes` feature inside of `docker-compose.yml`, what has been mentioned in https://github.com/woodpecker-ci/helm/pull/47 ?

**Comment by pat-s** [comment]
Helm is not docker-compose. You can mount as many volumes as you like in docker-compose (?).

**Comment by GamePlayer-8** [comment]
how can I do this for temporary containers made by woodpecker agent?
